src/core/Microsoft.PowerToys.Settings.UI/Views/ShortcutGuidePage.xaml

```diff
@@ -101,26 +102,37 @@
             </muxc:RadioButtons>
         </StackPanel>
 
-        <StackPanel 
-            x:Name="SidePanel"
-            Orientation="Vertical"
-            HorizontalAlignment="Left"
-            Width="{StaticResource SidePanelWidth}"
-            Grid.Column="1">
-
-            <TextBlock 
-                x:Uid="About_ShortcutGuide"
-                Style="{StaticResource SettingsGroupTitleStyle}"
-                Margin="{StaticResource XSmallBottomMargin}"
-                />
+        <RelativePanel x:Name="SidePanel" 
+                    HorizontalAlignment="Left"
+                    Width="{StaticResource SidePanelWidth}"
+                    Grid.Column="1">
+            <StackPanel x:Name="DescriptionPanel">
+                <TextBlock x:Uid="About_ShortcutGuide" x:Name="AboutTitle" Grid.ColumnSpan="2"
+                       Style="{StaticResource SettingsGroupTitleStyle}"
```

Comment:
lets try to remove the extra spaces (code formatting)

Reply:
Good catch, will fix